### PR TITLE
made a monorepo, added original solidiy package and added the Data Union subgraph package

### PR DESCRIPTION
**Pull request by samt1803**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,35 +1,22 @@
...
+    "clean": "npm run clean -ws",
+    "build": "npm run build -ws",
+    "lint": "npm run lint -ws",
+    "format-check": "npm run format-check -w=data-union-solidity",
+    "test": "npm run test -w=data-union-solidity",
+    "integration-test": "npm run integration-test -w=data-union-solidity",
+    "graph1_build": "npm run deploy -w=data-union-solidity",
+    "graph2_copyInfo": "./scripts/copyAbis.sh",
+    "graph3_build": "npm run doAll -w=streamregistry-thegraph-subgraph",
+    "graph": "npm run graph1_build && npm run graph2_copyInfo && npm run graph3_build"
```
Comment (jtakalai) [inline package.json:20]: Probably these steps aren't really necessary as separate?

File: package.json [hunk 2]
```diff
@@ -1,35 +1,22 @@
...
+  "workspaces": [
+    "./packages/*"
+  ],
+  "scripts": {
+    "clean": "npm run clean -ws",
+    "build": "npm run build -ws",
+    "lint": "npm run lint -ws",
+    "format-check": "npm run format-check -w=data-union-solidity",
+    "test": "npm run test -w=data-union-solidity",
+    "integration-test": "npm run integration-test -w=data-union-solidity",
```
Comment (jtakalai) [inline package.json:16]: I guess thegraph doesn't have tests yet?

**Review by jtakalai — APPROVED** [review]
combining DU subgraph into contracts repo LGTM

(maybe after merging, in a separate ticket?) following should still be done:
* rename repo to data-union-contracts
* clean up package.json
* add integration tests to thegraph part

File: package.json (hunk 2 above)
Comment (samt1803) [reply package.json:16]: there are no real unittests testing theGraph, in other repos theGraph is tested in integration-tests (but don't exist for the data-union either yet)

File: package.json (hunk 1 above)
Comment (samt1803) [reply package.json:20]: yes, they are copied from the network-contracts, just there for convenience or as a base for future scripting

**Pull request merged**